crates/iota-sdk-ffi/src/types/gas.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_types::GasCostSummary;
+
+#[uniffi::remote(Record)]
+pub struct GasCostSummary {
```

Comment:
What's happening here? Why do we have two symbols with the same name? Why aren't we wrapping?